aten/src/ATen/native/Repeat.h

```diff
@@ -17,7 +17,7 @@ template <
     void compute(index_t*, int64_t*, index_t*, int64_t, int64_t)>
 static inline Tensor repeat_interleave_common(
     const Tensor& repeats,
-    c10::optional<int64_t> output_size) {
+    c10::optional<SymInt> output_size) {
```

Comment:
This does not suffice - the `int64_t` below needs to be a c10::SymInt now, empty needs to be empty_symint, the `compute` template needs to be rewritten for symints, etc etc - you need to symintify the whole impl to match the body.